### ISSUE TITLE
Demo Data

### ISSUE THREAD

**Issue by jrycastillo** [SamoraHunter/Deep-Variational-Autoencoder-3D#1]
Hello, @SamoraHunter 

I am working with your code and I was able to do the training using my custom data.
The result of the training is not good because I'm getting extremely high loss.

Can I request for your data(csv) you used in the code?
I will just check if I will still get the same result.

Thank you in advance.

**Comment by SamoraHunter** [comment]
> 
@jrycastillo 

Thanks for trying my code. 
Unfortunately the raw data is not available at the moment. 

The script is also due to be tidied up soon. 

The dataset was all of the same "class" of object, with subtle but real differences/ variance in features/noise. The size of the dataset was 1000, with 3000 additional samples created by averaging 2-5 random samples from the true data. There were 11000 c. dimensions though only 6000 or so contained variance. 

I attach also a rudimentary parameter/architecture search script used to choose the number of layers. I also tweaked the level of compression which is propagated through the layers toward and from the latent dimension. The latent dimension size may also be tweaked. The learning rate too has a big impact. 

There is code to produce images of your generated meshes in pptk too to examine changes to parameters. 

*I'm not 100% sure about the calculation of the loss function as I tweaked it from an implementation found else where. 

Here is a small sample of the data to give an idea of how similar each sample is:
n.b some points across the 11k or so dimensions were identical for most samples too. 
*These are the xyzxyzxyz coordinates


<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/Artemis/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/Artemis/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">



4.040555 | -2.780733 | 113.262947 | 3.91311 | -2.532412 | 113.462578 | 3.578881 | -3.159334 | 113.793854 | 3.763192 | -3.216416 | 113.470428
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
4.040555 | -2.780733 | 113.262947 | 3.91311 | -2.532412 | 113.462578 | 3.578881 | -3.159334 | 113.793854 | 3.763192 | -3.216416 | 113.470428
4.064868 | -2.79262 | 113.261032 | 3.925778 | -2.539116 | 113.463196 | 3.592779 | -3.169648 | 113.794998 | 3.788126 | -3.229459 | 113.468956



</body>

</html>
